Readme.md

```diff
@@ -211,6 +211,19 @@ You can run any kind of code in parallel with -e / --execute
 <tr><td>Rails-ActionPack</td><td>88s</td><td>53s</td><td>44s</td></tr>
 </table>
 
+When scripting, the PARALLEL_TEST_GROUPS environment variable is also useful to know how many processes are running in parallel.
```

Comment:
How out adding this to the wiki, it's not really the default usecase to be meddling with internals like that ...